Bottom: f28ec49823485e069c95d7c5851dbd01583cf091
Top:    1f3e91359a2a6d34a157b3b986b2afac307079ae
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-08-17 14:02:45 -0700

librdmacm: Enable AF_IB support

Signed-off-by: Sean Hefty <sean.hefty@intel.com>


---

diff --git a/src/cma.c b/src/cma.c
index 5280620..9286e81 100755
--- a/src/cma.c
+++ b/src/cma.c
@@ -185,9 +185,6 @@ static void ucma_set_af_ib_support(void)
 	struct sockaddr_ib sib;
 	int ret;
 
-	/* Just return - do not enable AF_IB support for now */
-	return;
-
 	ret = rdma_create_id(NULL, &id, NULL, RDMA_PS_IB);
 	if (ret)
 		return;
